<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VistaInserisciSpettacoloAmministratore</class>
 <widget class="QWidget" name="VistaInserisciSpettacoloAmministratore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inserisci Spettacolo - CineMax</string>
  </property>
  <widget class="QLabel" name="Sfondo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>501</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background: qlineargradient(
        x1: 0, y1: 1,
        x2: 0, y2: 0,
        stop: 0 #320F00,
        stop: 1 #641E00
    );
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelIconaFotoAmministratore">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Immagini/IconaFotoAmministratore.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitolo">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Titolo:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitoloPrincipale">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Variable</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Inserisci Spettacolo</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelGenere">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Genere:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelOrarioInizio">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Orario Inizio:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelSala">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Sala:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelData">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Data:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelOrarioFine">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>350</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Orario Fine:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDurata">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Durata:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTitolo">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>291</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #320F00;
    color: #962D00;
    border: 1px solid #190700;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditGenere">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #320F00;
    color: #962D00;
    border: 1px solid #190700;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDurata">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>51</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #320F00;
    color: #962D00;
    border: 1px solid #190700;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConferma">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>450</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #961E00;
    color: #FF3C00;
    border: 2px solid #501400;
    border-radius: 6px;
}

QPushButton:hover {
    background-color: #C83200;
}</string>
   </property>
   <property name="text">
    <string>Conferma</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAnnulla">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>450</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #961E00;
    color: #FF3C00;
    border: 2px solid #501400;
    border-radius: 6px;
}

QPushButton:hover {
    background-color: #C83200;
}</string>
   </property>
   <property name="text">
    <string>Annulla</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxSala">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>76</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    background-color: #320F00;
    color: #962D00;
    border: 1px solid #190700;
    border-radius: 4px;
}

QComboBox QAbstractItemView {
    background-color: #320F00;
	color: #962D00;
    border: 1px solid #190700;
	border-radius: 4px;
    selection-background-color: #C83200;
}

QComboBox QAbstractItemView::item:hover {
    background-color: #C83200;
}</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditData">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>110</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateEdit {
    background-color: #320F00;
    color: #962D00;
    border: 1px solid #190700;
    border-radius: 4px;
	transition: border-color 1.5s ease;
}

QDateEdit:focus {
	border-color:#FF3C00;
	transition: border-color 1.5s ease;
}

</string>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>22</hour>
     <minute>0</minute>
     <second>0</second>
     <year>1950</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="time">
    <time>
     <hour>22</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
   <property name="maximumDateTime">
    <datetime>
     <hour>14</hour>
     <minute>59</minute>
     <second>59</second>
     <year>2050</year>
     <month>12</month>
     <day>12</day>
    </datetime>
   </property>
   <property name="minimumDateTime">
    <datetime>
     <hour>17</hour>
     <minute>0</minute>
     <second>0</second>
     <year>1949</year>
     <month>12</month>
     <day>31</day>
    </datetime>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEditOrarioInizio">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>118</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTimeEdit {
    background-color: #320F00;
    color: #962D00;
    border: 1px solid #190700;
    border-radius: 4px;
	transition: border-color 1.5s ease;
}

QTimeEdit:focus {
	border-color:#FF3C00;
	transition: border-color 1.5s ease;
}

</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEditOrarioFine">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>380</y>
     <width>118</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTimeEdit {
    background-color: #320F00;
    color: #962D00;
    border: 1px solid #190700;
    border-radius: 4px;
	transition: border-color 1.5s ease;
}

QTimeEdit:focus {
	border-color:#FF3C00;
	transition: border-color 1.5s ease;
}

</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelErroreTitolo">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>120</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FF0000
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelErroreGenere">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FF0000
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelErroreSala">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FF0000
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelErroreData">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FF0000
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelErroreOrarioInizio">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>380</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FF0000
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelErroreOrarioFine">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>380</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FF0000
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelErroreDurata">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>440</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FF0000
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrezzo">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>210</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Prezzo:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPrezzo">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>230</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #320F00;
    color: #962D00;
    border: 1px solid #190700;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrezzoPunti">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>270</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string> Prezzo in punti:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPrezzoPunti">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>290</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #320F00;
    color: #962D00;
    border: 1px solid #190700;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelErrorePrezzo">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>230</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FF0000
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelErrorePrezzoPunti">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>290</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FF0000
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
